python/mlc_chat/compiler/quantization/group_quantization.py

```diff
@@ -119,7 +178,7 @@ def quantize_weight(self, weight: NDArray) -> List[NDArray]:
             )(mod)
         ex = relax.build(mod, "cuda")
         dev = device("cuda", 0)
-        vm = relax.VirtualMachine(ex, dev)
+        vm = relax.VirtualMachine(ex, dev)  # pylint: disable=invalid-name
         return vm["quantize"](weight)
```

Comment:
```suggestion
        return vm["quantize"](weight.copyto(dev))
```

Reply:
Fixed by using `weight.device` to build and quantize.